tests/nnfw_api/src/one_op_tests/Softmax.cc

```diff
@@ -62,6 +62,38 @@ TEST_P(SoftmaxVariation, Test)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_Softmax)
+{
+  CircleGen cgen;
+  int lhs = cgen.addTensor({{4, 1, 1, 1}, circle::TensorType::TensorType_FLOAT32});
+  int out = cgen.addTensor({{4, 1, 1, 1}, circle::TensorType::TensorType_FLOAT32});
+  cgen.addOperatorSoftmax({{lhs}, {out}}, 1.0);
+  cgen.setInputsAndOutputs({lhs}, {out});
+
+  _context = std::make_unique<GenModelTestContext>(cgen.finish());
+  _context->addTestCase(uniformTCD<float>(
+    {{-1., 0., 1., 1.}}, {{1, 2.7182817459106445, 7.3890557289123535, 7.3890557289123535}}));
+  _context->setBackends({"gpu_cl"});
+
```

Comment:
I'm tring to test this pr in my enviroment.
But in `setBackends` function, it doesn't seems to set gpu_cl backend.
because `TEST_GPU_CL_BACKEND` wasn't defined ..
how can i enable `TEST_GPU_CL_BACKEND` ?

Reply:
@hasw7569 You need https://github.com/Samsung/ONE/commit/e93bcb128a564a7b7224d7756e2bca9ba7bf94ad

Reply:
@chunseoklee Thank you : )